Src/DroneCAN/DroneCAN.c

```diff
@@ -1020,7 +1020,11 @@ static void send_ESCStatus(void)
     current.count = 0;
 
     pkt.temperature = C_TO_KELVIN(degrees_celsius);
-    pkt.rpm = (e_rpm * 200) / eepromBuffer.motor_poles;
+    if (forward) {
```

Comment:
```suggestion
    if (forward != eepromBuffer.dir_reversed) {
```

Tested on top of #164. Reversing the direction in the ESC shouldn't reverse the sign of the RPM. The RPM sign should match the throttle sign if the motor is operating correctly.

Reply:
@tpwrules can you make a PR for this, I am not sure how the negative RPM is actually used in FC(just CW/CCW in here), maybe you PR this will be better?